implementations/micrometer-registry-stackdriver/src/test/java/io/micrometer/stackdriver/StackdriverMeterRegistryTest.java

```diff
@@ -196,4 +196,19 @@ void meanIsZeroWhenCountIsZero() {
         assertThat(distribution.getMean()).isZero();
     }
 
+    @Test
+    @Issue("#6401")
+    void distributionWithNoBucketsUsesStepCount() {
+        StackdriverMeterRegistry.Batch batch = meterRegistry.new Batch();
+        // halfway through first step
+        clock.add(config.step().dividedBy(2));
+        DistributionSummary ds = DistributionSummary.builder("ds").register(meterRegistry);
+        ds.record(3);
+        // 1/4 through the second step
+        clock.add(config.step().dividedBy(4).multipliedBy(3));
+        Distribution distribution = batch.distribution(ds.takeSnapshot(), false);
+        System.out.println(distribution);
+        assertThat(distribution.getCount()).isEqualTo(1);
```

Comment:
Coincidentally the distribution count is equal to the step count here. It may be better to test this in a setup when they wouldn't be equal.